package.json

```diff
@@ -25,7 +25,7 @@
   "dependencies": {
     "prop-types": "^15.5.10",
     "react-native-tabular-grid": "github:Benjamin-Dobell/react-native-tabular-grid",
-    "simple-markdown": "^0.2.1"
+    "simple-markdown": "0.3.x"
```

Comment:
There aren't relevant changes in 0.3.0 for this, but 0.3.0 fixes some bold/italics nesting issues, which we might as well fix while we're here.